File: Dataset/Leetcode/train/70/218.java
class Solution {
    public int XXX(int n) {
        if(n==1){
            return 1;
        }else if(n==2){
            return 2;
        }else{
            int s1 = 1;
            int s2 = 2;
            int si = 0;
            for(int i=3;i<=n;i++){
                si= s1+s2;
                s1=s2;
                s2=si;
            }
            return si;
        }
        
    }
}

File: Dataset/Leetcode/train/75/218.cpp
class Solution {
public:
    void XXX(vector<int>& nums) {
            int n = nums.size();
    auto it = nums.begin();
    int indx =0;
    for (int i = 0; i < n; i++)
    {
        if (*it == 0)
        {
            nums.insert(nums.begin(), 0);
            it = nums.begin() + i+1-indx;
            it = nums.erase(it);
        }
        else if (*it == 2)
        {
            it = nums.erase(it);
            nums.push_back(2);
            indx++;
        }
        else
        {
            it++;
        }
    }
    }
};

